app/src/main/java/org/traccar/client/TrackingService.kt

```diff
@@ -90,7 +92,9 @@ class TrackingService : Service() {
     override fun onDestroy() {
         stopForeground(true)
         Log.i(TAG, "service destroy")
-        sendBroadcast(Intent(ACTION_STOPPED))
+        // Explicit package name is required here for manifest-declared receiver of the status widget
+        // Refer to https://developer.android.com/guide/components/broadcasts#manifest-declared-receivers
```

Comment:
I don't like this duplicated comment. Can we move this to the class level comment?